data_collection/gazette/spiders/sp_paulinia.py

```diff
@@ -0,0 +1,39 @@
+from urllib.parse import urljoin
+
+import dateparser
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+START_URL = "http://www.paulinia.sp.gov.br/semanarios"
+
+
+class SpPauliniaSpider(BaseGazetteSpider):
+    name = "sp_paulinia"
+    TERRITORY_ID = "3536505"
+
+    start_urls = [START_URL]
+
+    def parse(self, response):
+        for year_href in response.css('a[id^="corpo_lnkItem"]::attr(href)').extract():
+            year_value = year_href.split("'")[1]
+            yield scrapy.FormRequest.from_response(
+                response,
+                formdata={"__EVENTTARGET": year_value},
+                callback=self.parse_gazettes,
+            )
+
+    def parse_gazettes(self, response):
+        for link in response.css('a[href^="AbreSemanario"]'):
+            datestamp, edition, edition_type = (
+                link.css("::text").get().strip().split(" - ")
```

Comment:
When testing, I got one error in this line, when debugging I found that the returned link was `07/12/2020 - 1565- Edição Normal  `, the problem was in the `1565-` where you are expecting a space after the number, can you handle this edge case?